bodhi/tests/server/functional/test_generic.py

```diff
@@ -342,6 +343,22 @@ def test_latest_builds(self):
         self.assertIn('f17-override', body)
         self.assertEquals(body['f17-updates'], 'TurboGears-1.0.2.2-2.fc17')
 
+    @mock.patch("bodhi.server.buildsys.DevBuildsys.getLatestBuilds", side_effect=Exception())
+    def test_latest_builds_exception(self, mock_getlatestbuilds):
+        """
+        Test that the latest_builds() just passes if it hits an exception
+        the result here is that there should be no builds returned
+        """
+        res = self.app.get('/latest_builds')
+        body = res.json_body
+        print body
```

Comment:
Let's drop this print statement.

Reply:
gah! thanks for spotting -- it seems i am destined to always leave one of these lying around!